loc.gov JSON API/Chronicling_America_Title_Essay_Datasets/download.ipynb

```diff
@@ -0,0 +1,2007 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "c12f4cc3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "links = []\n",
+    "for page in pages:\n",
+    "    response = requests.get(page)\n",
+    "    time.sleep(2)\n",
+    "    soup = BeautifulSoup(response.text, \"html.parser\")\n",
+    "    for title in soup.find_all(\"span\", \"item-description-title\"):\n",
+    "        link = title.find(\"a\")[\"href\"]\n",
+    "        links.append(link)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "014dab94",
+   "metadata": {},
+   "source": [
+    "Let's list the first 10 links to make sure everything worked correctly:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "54c3e860",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['https://www.loc.gov/item/sn85026945/',\n",
+       " 'https://www.loc.gov/item/sn93067670/',\n",
+       " 'https://www.loc.gov/item/sn93067668/',\n",
+       " 'https://www.loc.gov/item/sn84026853/',\n",
+       " 'https://www.loc.gov/item/sn85042527/',\n",
+       " 'https://www.loc.gov/item/sn88064057/',\n",
+       " 'https://www.loc.gov/item/sn83045004/',\n",
+       " 'https://www.loc.gov/item/sn83045003/',\n",
+       " 'https://www.loc.gov/item/sn98069055/',\n",
+       " 'https://www.loc.gov/item/sn83016734/']"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "links[:10]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d99a3941",
+   "metadata": {},
+   "source": [
+    "And that those link out to a single column csv for analysis and storage."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "d3b45720",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "header = [\"lccn\"]\n",
+    "with open(\"lc_output.csv\", \"w\") as f:\n",
+    "    write = csv.writer(f) \n",
+    "    write.writerow(header) \n",
+    "    for link in links:\n",
+    "        write.writerow([link])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "386884d0",
+   "metadata": {},
+   "source": [
+    "------------"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7ab616c2",
+   "metadata": {},
+   "source": [
+    "## Part 2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "174f57cb",
+   "metadata": {},
+   "source": [
+    "---------"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "79cf4c70",
+   "metadata": {},
+   "source": [
+    "Next, we will take that list of URLs and create a small script to pull down data from the Library API. I originally wrote it as a parallelized script that downloaded the code in about 45 minutes. Unfortunately, it kept overloading the Library of Congress API. I re-wrote it so that it instead uses a handful of 'user-agents' to download the data. This \"slow\" method takes about three hours. "
```

Comment:
We'd appreciate it if you removed this section on using multiple user agents (Part 2 first markdown cell, ll. 266-306,  and input 11 code cell through input 13 code cell, ll. 479-539), replace the call for `rando_headers()` in l. 1526 with a note on how to set a custom header, and increased the value in `time.sleep(0.8)` in l. 1548 to match the rate limits on https://www.loc.gov/apis/json-and-yaml/ for the newspapers endpoint.

Reply:
Thanks for comments! Should be resolved. Let me know if this fixes issues.